graph/src/data/store/mod.rs

```diff
@@ -439,6 +440,75 @@ impl<'a> From<Vec<(&'a str, Value)>> for Entity {
     }
 }
 
+pub trait TryFromEntity: Sized {
```

Comment:
@leodasvacas I'm particularly curious if you have better suggestions for designing the helper methods here.